python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -476,9 +496,9 @@ class DecorrelateALKernelSpatialConfig(pexConfig.Config):
     )
 
     def setDefaults(self):
-        self.decorrelateMapReduceConfig.gridStepX = self.decorrelateMapReduceConfig.gridStepY = 19
-        self.decorrelateMapReduceConfig.cellSizeX = self.decorrelateMapReduceConfig.cellSizeY = 20
-        self.decorrelateMapReduceConfig.borderSizeX = self.decorrelateMapReduceConfig.borderSizeY = 6
+        self.decorrelateMapReduceConfig.gridStepX = self.decorrelateMapReduceConfig.gridStepY = 40
+        self.decorrelateMapReduceConfig.cellSizeX = self.decorrelateMapReduceConfig.cellSizeY = 41
+        self.decorrelateMapReduceConfig.borderSizeX = self.decorrelateMapReduceConfig.borderSizeY = 8
```

Comment:
This code appears to be repeated below (line 1042). Can it be centralized?

Reply:
I think you mean it's also in the `zogy.py` code. I can centralize it as the default in the `imageMapReduce.ImageMapReduceConfig`. However, there might be a reason to have these be different in the future. Right now I'm just finding that the same values work for both tasks. Do you have a suggestion about where to centralize this?

Reply:
It's a good question. It *looks* like the defaults here actually belong to the config `self.decorrelateMapReduceConfig`, which is  a `DecorrelateALKernelMapReduceConfig`, rather than this config. If so, and if that is true for `zogy.py` then I suggest those two map-reduce config classes inherit from a new common base class (e.g. `BaseMapReduceConfig`) that defines these common fields and assigns suitable defaults. Then in the future if one wishes to depart from any of these defaults, one can do that in the appropriate subclass or here.

For this to work `setDefaults` will have to call setDefaults on contained configs -- which is a good thing to do in any case.

However, such a change could be done as part of the refactoring. It's just the since you had to change the values in 2 places, that's a red flag.